test/functional/api/v2/compliance/arf_reports_controller_test.rb

```diff
@@ -9,7 +9,13 @@ class Api::V2::Compliance::ArfReportsControllerTest < ActionController::TestCase
     @report = FactoryGirl.create(:arf_report,
                                  :host_id => @host.id,
                                  :openscap_proxy => FactoryGirl.create(:smart_proxy, :url => "http://smart-proxy.org:8000"))
+    @policy = FactoryGirl.create(:policy)
+    @asset = FactoryGirl.create(:asset)
+
+    arf_string = File.open "#{ForemanOpenscap::Engine.root}/test/files/arf_report/arf_report.json", &:read
```

Comment:
I think this is equivalent to `File.read "#{ForemanOpenscap::Engine.root}/test/files/arf_report/arf_report.json"`